src/icons/Arrow.tsx

```diff
@@ -36,6 +36,8 @@ export default function ({
   openDisabled,
   open,
 }: ArrowProps) {
+  const strokeId = Math.random().toString()
```

Comment:
Maybe I need to leave a comment here, basically if you have 2 elements with same `id` the parser will take the upper one. We use `Arrow` icon in mobile `Dropdown` and we hide it, for some reason another arrow icon (email proof) will try to use non-existing `stroke="url(gradientXxx)"`, so it will be transparent, to prevent this, we need uniaue id for each svg-defs